pkgs/servers/web-apps/pgpkeyserver-lite/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, lib } : 
+
+stdenv.mkDerivation rec {
+  name = "pgpkeyserver-lite-${version}";
+  version = "2017-07-18";
+
+  src = fetchFromGitHub {
+    owner = "mattrude";
+    repo = "pgpkeyserver-lite";
+    rev = "a038cb7";
+    sha256 = "12pn92pcpv38b2gmamppn9yzdn7x52pgxnzpal22gqsxwimhs2rx";
+  };
+
+  installPhase = ''
+    mkdir -p $out
+    cp -R 404.html assets favicon.ico index.html robots.txt $out
+  '';
+
+  meta = {
```

Comment:
not sure about the official guidelines here, but AFAIK it's always preferred to add yourself as maintainer when adding a new package to the package set.